apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -290,8 +290,7 @@ groups() ->
        {group, swagger_validation},
```

Comment:
I see in this file there is leftover legacy - please see patch in #2391 (untested).